content/docs/secure/encryption/rekey.md

```diff
@@ -1,24 +1,25 @@
 ---
 title: "Rekey Configuration"
 linkTitle: "Rekey Configuration"
-weight: 3
+weight: 4
 Description: >
   Rekey Configuration and Usage
 ---
 
 ## Rekey Controller Installation
 
-The CSM Encryption Rekey Controller must be installed in order to be able to perform key rotation on CSM encrypted volumes in a
-Kubernetes cluster. The Rekey Controller can be installed via the standard Dell Helm Chart repository available at  https://github.com/dell/helm-charts.git.
+The CSM Encryption Rekey CRD Controller is an optional component that, if installed, allows encrypted volumes rekeying in a
+Kubernetes cluster. The Rekey Controller can be installed via the standard Dell Helm Chart repository available 
+at https://github.com/dell/helm-charts.
 
 A secret of the cluster config must be created with the name ``cluster-kube-config`` typically from the .kube/config. Here is an example:  
 
+```shell
+ kubectl create secret generic cluster-kube-config --from-file=/home/root/.kube/config
 ```
-kubectl create secret generic cluster-kube-config --from-file=/home/root/.kube/config
-```
 
-The `csm-encryption-rekey-controller/values.yaml` must be adapted to the kubernetes cluster via the `provisioner` and `port` parameters.
-Here is an example of a typical values.yaml file:
+The Rekey Controller Helm chart defines these values `csm-encryption-rekey-controller/values.yaml`:

```

Comment:
Please remove " `csm-encryption-rekey-controller/values.yaml`". All charts define their values in values.yaml, no need to highlight this. And the directory name may be any.

Reply:
removed.